src/ffjord.jl

```diff
@@ -0,0 +1,56 @@
+abstract type CNFLayer <: Function end
+Flux.trainable(m::CNFLayer) = (m.p,)
+
+struct FFJORD{M,P,RE,ContinuousUnivariateDistribution,Bool,T,A,K} <: CNFLayer
+    model::M
+    p::P
+    re::RE
+    baseDist::ContinuousUnivariateDistribution
+    MonteCarlo::Bool
+    tspan::T
+    args::A
+    kwargs::K
+
+    function FFJORD(model,tspan,args...;p = nothing,baseDist=nothing,MonteCarlo=nothing,kwargs...)
```

Comment:
```suggestion
    function FFJORD(model,tspan,args...;p = nothing,basedist=Normal(0,1),monte_carlo=false,kwargs...)
```

Following the standard naming conventions for Julia variables

Reply:
Ok. Should I also change the fields in the struct to follow the convention?

Reply:
yes